doc/dev/tests.md

```diff
@@ -346,7 +356,7 @@ not locally available.
 ##### Environment prerequisites
 
 - The targeted library is already migrated to use the test proxy.
-- Git version > 2.30.0 is to on the machine and in the path. Git is used by the script and test proxy.
+- Git version > 2.30.0 installed on the machine and in the path. Git is used by the script and test proxy.
```

Comment:
Thanks for the catch! This was originally supposed to be "is on", but "is installed on" is a fine fix
```suggestion
- Git version > 2.30.0 is installed on the machine and in the path. Git is used by the script and test proxy.
```